jobs/firehose_exporter/templates/bin/firehose_exporter_ctl

```diff
@@ -40,8 +40,12 @@ case $1 in
       --uaa.client-secret="<%= p('firehose_exporter.uaa.client_secret') %>" \
       --doppler.url="<%= p('firehose_exporter.doppler.url') %>" \
       <% if_p('firehose_exporter.doppler.subscription_id') do |subscription_id| %> \
+      <% if p('firehose_exporter.doppler.use_unique_subscription_id') %> \
```

Comment:
We tend to use the `if_p` pattern when the property does not have a default value. In this PR, you're adding a `prometheus` default value to the property, so better to remove [L42](https://github.com/bosh-prometheus/prometheus-boshrelease/pull/291/files#diff-0893ddf0c255e5596e264a38959eec2bR42) and use directly the property:

```
<% if p('firehose_exporter.doppler.use_unique_subscription_id') %> \
--doppler.subscription-id="<%= "#{p('firehose_exporter.doppler.subscription_id')}-#{spec.id}" %>" \
<% else %> \
--doppler.subscription-id="<%= p('firehose_exporter.doppler.subscription_id') %>" \
<% end %> \
```

Reply:
You're right, fixed.